### DISCUSSION TITLE
How does one use the memory balloon?

### DISCUSSION THREAD

**Discussion by Atemu** [utmapp/UTM#4797] [UTM Help (macOS)]
I can enable it and the guest sees the PCI device but it doesn't seem to do anything?

I've tried adding `-m size=1G,maxmem=8G` but it doesn't grow past 1G.

I want to have 1G to the VM by default but want it to be able to grow to i.e. 8G if it needs the memory for applications but not pollute the memory with excessive inode cache.

**Comment by agudelodiego** [comment]
?

**Reply by KendrickLamar2003** [reply]
what the fuck

**Reply by KendrickLamar2003** [reply]
We are currently being invaded by bots it seems like
